shardingsphere-distribution/shardingsphere-scaling-distribution/pom.xml

```diff
@@ -45,7 +45,13 @@
             <artifactId>postgresql</artifactId>
             <scope>runtime</scope>
         </dependency>
-    
+
+        <dependency>
+            <groupId>org.opengauss</groupId>
+            <artifactId>opengauss-jdbc</artifactId>
+            <scope>provided</scope>
+        </dependency>
+
```

Comment:
It might be not required, since `shardingsphere-scaling-bootstrap` is a dependency.